selectors/Overview.bs

```diff
@@ -2031,6 +2036,49 @@ The Input Focus Pseudo-class: '':focus''</h3>
 	the same ways must apply to elements matching '':focus'' as well,
 	except that the parent of an element that matches '':focus'' must not match '':focus''.
 
+<h3 id="the-focusring-pseudo">
+The Input Focusring Pseudo-class: '':focusring''</h3>
+
+	The <dfn id="focusring-pseudo'>:focusring</dfn> pseudo-class is similar to the
+	:focus pseudo-class, but it only matches an element if the element is
+	currently focused and a focus ring or other indicator should be drawn for that
+	element. If :focusring matches, then :focus also matches, but the converse is
+	not always true - it depends on whether the user agent has focus ring drawing
+	enabled and how the element was focused. Whether the user agent has focus ring
+	drawing enabled can depend on things like the settings of the operating system
+	the user is using, so the precise behavior of this pseudo-class can vary from
+	platform to platform depending on each platform's particular focus best
+	practices (defaults) or user modified settings.
+
+	<div class='example'>
+		For example, most UAs choose to display focus rings on native text fields
+		whenever they are focused, and on native buttons only when selected via
+		the keyboard.
+	</div>
+
+	<em>This section is non-normative, and is meant to help clarify when a
+	:focusring selector might match.</em>
+
+	A user agent might choose to use the following heuristic for deciding when to
+	match '':focusring'' on focused elements which do not have any native focus
+	ring behaviour defined:
+
+	When focusing an element:
+	*   if the element received focus via a keyboard interaction, including
+	indirectly such as triggering a dialog via pressing a button using the
+	keyboard, apply '':focusring''.
+
+	When handling a keyboard event:
+	*   if a non-native element is focused, even if it did not become focused via
+	the keyboard, apply '':focusring'' to that element.
+
+	<div class='note'>
```

Comment:
double checking the spec I found examples of both "note" and 'note'. happy to do it all as double quotes though to make it consistent in the PR